ci/run_clang_tidy.sh

```diff
@@ -1,9 +1,16 @@
-#!/bin/bash
+#! /usr/bin/env bash
```

Comment:
nit: extra space after `!`

Reply:
Space here is acceptable, but I can remove if you prefer :)

Reply:
I know it is acceptable, but prefer not having it for the consistency

Reply:
Fixed.